internal/main.go

```diff
@@ -66,10 +66,11 @@ func main() {
 		fail("failed to read %s: %v\n", infile.Name(), err)
 	}
 
-	dataOut, err := config.Translate(dataIn, options)
+	dataOut, r, err := config.Translate(dataIn, options)
 	if err != nil {
 		fail("Error translating config: %v\n", err)
 	}
+	fmt.Fprintf(os.Stderr, "%s", r.String())
```

Comment:
Should the format string include a line break?

Reply:
No. `r.String` should be `""` when there's nothing in the report, but should include a trailing newline otherwise. If we put one in we'd double-print.